tests/std/tests/P1206R7_vector_from_range/test.cpp

```diff
@@ -161,6 +161,12 @@ void test_vso1591034() {
     assert(counted_item::count == 6);
 }
 
+void test_LWG_3743() { // COMPILE-ONLY
+    ranges::take_view t       = views::iota(0ULL) | views::take(5);
+    ranges::subrange r        = ranges::subrange(t, 5);
+    [[maybe_unused]] vector v = r | ranges::to<vector<unsigned long long>>(size_t{0});

```

Comment:
1. `T x = T(args)` is a bit weird.
2. If we don't name the result vector, we don't need to say it's unused.
3. We've lost all resemblance to the example in the issue, we may as well give it up completely.
```suggestion
    ranges::subrange r(views::iota(0ULL) | views::take(5), 5);
    (void)ranges::to<vector<unsigned long long>>(r, size_t{0});
```